store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -470,11 +526,29 @@ public CarbonWriter buildThreadSafeWriterForAvroInput(org.apache.avro.Schema avr
     return new AvroCarbonWriter(loadModel, configuration);
   }
 
+  /**
+   * Build a {@link CarbonWriter}, which accepts Avro object
+   *
+   * @param avroSchema   avro Schema object {org.apache.avro.Schema}
+   * @param numOfThreads number of threads() in which .write will be called.
+   * @return AvroCarbonWriter
+   * @throws IOException
```

Comment:
since you are modifying this, please complete all comment

Reply:
ok